docs/online/src/scenarios/postmessage.rst

```diff
@@ -74,11 +74,44 @@ messages you send to its iframe.
 
 You can send the following messages; all others are ignored:
 
+* :data:`App_PopState`
 * :data:`Blur_Focus`
 * :data:`Grab_Focus`
 * :data:`Host_PerfTiming`
 * :data:`Host_PostmessageReady`
 
+..  data:: App_PopState
+
+    The App_PopState message signals the Office Online application that state has been popped from the HTML5 History
+    API to which the application should navigate to using the URL. This message should be triggered from an
+    `onpopstate` listener in the host page.
+
+    ..  attribute:: Values
+        :noindex:
+
+        Url *(string)*
+            The URL associated with the popped history state.
+    
+        State *(JSON-formatted object)*
+            The data associated with the state.
+
+    ..  rubric:: Example Message:
+
+    ..  code-block:: JSON
+
+        {
+            "MessageId": "App_PopState",
+            "SendTime": 1329014075000,
+            "Values": {
+                "Url": "https://www.contoso.com/abc123/contents?wdtarget=pagexyz",
+                "State": {
+                    "Value": 0
+                }
+                "wdUserSession": "3692f636-2add-4b64-8180-42e9411c4984",
```

Comment:
wdUserSession and ui-language should be removed in this message sent to Office Online